### DISCUSSION TITLE
How do I get the all-time playtime?

### DISCUSSION THREAD

**Discussion by TomasGnG** [plan-player-analytics/Plan#2864] [APIs and Development]
Where is the playtime stored in the db?
I want to code a Top 10 most active player Leaderboard but its difficult when I dont even know where plan stores the playtime.
I need the table to I can use SQL commands.

**Comment by AuroraLS3** [comment]
Playtime is in `plan_sessions`
```SELECT SUM(session_end-session_start) as playtime FROM plan_sessions;```

**Reply by TomasGnG** [reply]
Thanks for the reply but do you know if there is a method to update the query instance? Because everytime I use `queryService.getCommonQueries().fetchPlaytime()` the value stays the same untill I rejoin

**Reply by AuroraLS3** [reply]
The online playtime can be taken with the `queryService.getCommonQueries().fetchCurrentSessionPlaytime()` :)

The playtime in database is only updated when player leaves since online sessions are not yet stored there

**Reply by TomasGnG** [reply]
Love you man! :D
